deep-learning/_posts/2017-09-17-3dbiomed-segmen.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: review
+title:  "Joint Sequence Learning and Cross-Modality Convolution for 3D Biomedical Segmentation"
+tags:   3D Biomedical Segmentation
```

Comment:
Please only put tags mentioned in https://vitalab.github.io/howto/

Also, put "segmetation" and "CNN"